cmd/coder/shell.go

```diff
@@ -142,7 +143,12 @@ func runCommand(ctx context.Context, envName string, command string, args []stri
 	go func() {
 		stdin := process.Stdin()
 		defer stdin.Close()
-		io.Copy(stdin, os.Stdin)
+
+		ap := &activity.Pusher{Source: "ssh", EnvID: env.ID, Client: entClient}
+		defer ap.Start()()
+
+		wr := activity.Writer(ap, stdin)
```

Comment:
Why not put the Writer method on the Activity Pusher? We get better autocomplete discoverability and a shorter invokation:

```suggestion
		wr := ap.Writer(stdin)
```